brain/processor.py

```
@@ -43,12 +44,32 @@ async def process(self, query: str, context: Optional[Dict[str, Any]] = None) ->
             " For tool_calls, be specific about which tools from the server would be used (e.g., ['save_memory', 'retrieve_memory'])."
         )
         user = f"Capabilities:\n{tools_brief}\n\nTask: {query}"
-        llm = self._get_llm()
-        resp = await llm.ainvoke([
-            {"role": "system", "content": system},
-            {"role": "user", "content": user},
-        ])
-        text = resp.content.strip()
+        
+        # Retry策略：重试2次，间隔1秒、2秒
```

Ruff (0.14.6)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)